server.js

```diff
@@ -123,6 +123,21 @@ app.get(/\.html?$/i, function (req, res){
   res.redirect(path);
 });
 
+// redirect if `redirect(<URL>)` is part of the submitted data.
+app.use('*',function(req,res,next)
+{  
+  var data;  
+  if (Object.keys(req.query).length !== 0) { data = req.query; }
+  else if (Object.keys(req.body).length !== 0) { data = req.body; }
+  if (data !== undefined) {
+    for (var key in data) {
+      var match = data[key].match(/^redirect\(([^\)]*)\)/);
+      if (match !== null ) res.redirect(match[1]);
```

Comment:
I think it would be good to log to console when you do the redirect - will make diagnosing faults easier.